Procfile.dev

```diff
@@ -1,3 +1,4 @@
 web: PORT=3000 bundle exec puma -C config/puma.rb
+sidekiq: bundle exec sidekiq -q default -q push -q pull -q mailers
```

Comment:
ah, sorry, this is my mistake. I misread this line and thought you were adding a queue here.